pack/side/tmg_encounter.json

```diff
@@ -0,0 +1,1411 @@
+[
+  {
+    "back_text": "Hard / Expert\n[skull]: -X. X is 1 plus the current agenda number.\n[cultist]: -X. X is the number of story assets at your location.\n[tablet]: -3. (-5 instead if you are at a [[Private]] location).\n[elder_thing]: -4. If you fail or if The Bloodless Man is at your location, take ? horror.",
```

Comment:
I could not make out the value of `?` in the card text.

Reply:
It's 1 horror